cmd/helm/release.go

```diff
@@ -35,5 +39,30 @@ func releaseCmd() cli.Command {
 				Usage: "Destination URL to which this will be POSTed.",
 			},
 		},
+		Action: func(c *cli.Context) { run(c, release) },
+	}
+}
+
+func release(c *cli.Context) error {
+	a := c.Args()
+	if len(a) == 0 {
+		return errors.New("'helm release' requires a path to a chart archive or directory.")
 	}
+
+	var arch string
```

Comment:
This block could use a pinch of refactor to help with readability.  Not a blocker for the PR